src/models/registerService.test.js

```diff
@@ -0,0 +1,49 @@
+import MockAdapter from 'axios-mock-adapter'
+const { registerUser } = await import('./registerService');
+import axios from 'axios';
+import {jest} from '@jest/globals'
+
+process.env.BASE_API_URL = 'http://localhost:8080';
+
+describe('Register User model', () => {
+  it('Should return user data upon successfully registering', async () => {
+    // --------
+    // ASSEMBLE
+    // --------
+    global.sessionStorage = {
+      setItem: jest.fn(),
+    };
```

Comment:
You don't need to mock that, because you are not using `sessionStorage` in your `registerUser` func